src/Generator.php

```diff
@@ -64,11 +63,8 @@ public function calcMod10V1($number, $separator = "", $minimumLength = 6)
      * @param string $number
      * @return string
      */
-    public function calcMod10V5($number)
+    public function calcMod10V5(string $number) : string
```

Comment:
There shouldn't be a space between the closing parenthesis and the colon.

Reply:
Resolved.